tests/test_validation.py

```diff
@@ -18,8 +18,45 @@
 from osgeo import ogr
 from osgeo import osr
 
+from natcap.invest import spec
```

Comment:
Do we need this import if you're specifying components below?

Reply:
Other cases of `spec` might be locally imported to the tests themselves?

Reply:
We can remove the redundant imports from the old test cases. AFAIK there is no reason to re-import the module within each test function. I think we have those extra imports left over from a time when it did matter?

Reply:
Yea, from my understanding if we're using the import everywhere then importing at top level makes sense. If only a few test cases used an import I could see isolating that import to those functions specifically.